src/observable.js

```diff
@@ -114,6 +114,45 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  observe(observer, onError, onComplete) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: onError,
+        complete: onComplete
+      };
+    }
+
+    const fn = function(event) {
+      if (event.type === VALUE && observer.next) {
+        observer.next(event.value);
+      } else if (event.type === ERROR && observer.error) {
+        observer.error(event.value);
+      } else if (event.type === END && observer.complete) {
+        observer.complete(event.value);
+        closed = true;
+      }
+    };
+
+    this.onAny(fn);
+
+    return {
+      unsubscribe() {
+        if (!closed) {
+          _this.offAny(fn);
+
+          closed = true;
+        }
+      },
+      get closed() {
```

Comment:
[IE9+](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get). Do you need IE8 support?